CHANGELOG.md

```diff
@@ -3,6 +3,95 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.33](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.32...v4.0.0-alpha.33)
+
+_July 1, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 💅 Allow to copy the selected rows to the clipboard with <kbd>CTRL</kbd> + <kbd>c</kbd> (#1929) @m4theushw
+- 🐛 `Select all` checkbox click should select only filtered rows (#1879) @ZeeshanTamboli
+- ⚡️ Add a new `singleSelect` column type (#1956) @DanailH
+
+  Using the column `type: 'singleSelect'` defaults to `Select` component when the cell is in `edit` mode. You can find the documentation [following this link](https://material-ui.com/components/data-grid/columns/#column-types).
+
+  ```tsx
```

Comment:
We never used `tsx` or `jsx` in the CHANGELOG. Maybe leave it for the release only.